ci/prow/boskos/config_start.yaml

```diff
@@ -21,3 +21,34 @@ metadata:
   namespace: test-pods
 data:
   resources: ""
+---
+kind: ClusterRole
```

Comment:
It looks like there's similar role binding in the general config_start.yaml. Do those need to be removed? 
https://github.com/knative/test-infra/blob/master/ci/prow/config_start.yaml#L56

Reply:
Good catch, thanks. Updated the right section in the right file.